lib/features/occupation/data/models/occupation_model.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:coffeecard/features/occupation/domain/entities/occupation.dart';
+import 'package:coffeecard/generated/api/coffeecard_api.models.swagger.dart';
+import 'package:coffeecard/generated/api/coffeecard_api_v2.models.swagger.dart';
+
+class OccupationModel extends Occupation {
```

Comment:
The point is that the model and entity are in two different layers. The entity exists within the domain and should only be used here. The model is a 'dirty' version of the entity that implements data-related logic such as parsing from API responses. Given this separation you can modify data mapping without affecting the domain model

Reply:
Not sure that I agree that it gives you any greater separation, since you are inheriting from the entity anyway, at least in this instance. Perhaps there are other cases where the advantage would be more obvious, but I digress. In the interest of moving on with the PR, let's leave it "as is" for now.